src/PrestaShopBundle/Install/Upgrade.php

```diff
@@ -303,6 +303,8 @@ private function initContext()
             if (!isset(Context::getContext()->language) || !Validate::isLoadedObject(Context::getContext()->language)) {
                 if ($id_lang = (int) $this->getConfValue('PS_LANG_DEFAULT')) {
                     Context::getContext()->language = new Language($id_lang);
+                } else {
+                    Context::getContext()->language = new Language();
```

Comment:
easier to read and refactor
```php
$langId = (int) $this->getConfValue('PS_LANG_DEFAULT');
Context::getContext()->language = new Language(
    $langId ?
    $langId :
    null
);
```

Reply:
is it really necessary to write on three lines?
```php
Context::getContext()->language = new Language($langId ? $langId : null);
```
the one line style is readable as well no?